### ISSUE TITLE
[DONATION] ImaginedOpenCloseFistFeet

### ISSUE THREAD

**Issue by AidenRushbrooke** [uea-machine-learning/tsml_repo#100]
ImaginedOpenCloseFistFeet
https://physionet.org/content/eegmmidb/1.0.0/
Collected as part of a larger motor imagery archive, this dataset consists EEG readings of imagined hand vs foot
movements. 105 individuals took part in the experiment, completing 45 trials each, imagining moving either both hands or both feet 
based on the location of a target on a screen. Each instance lasts 4 seconds starting 1 second before the movement began,
 and was recorded at 160Hz with 64 EEG channels. The first 52 participants have been used for the training set, and the remaining 53 for the test set.
